vector-im/element-x-android
# [Rich text editor] Ensure keyboard opens for reply and text formatting modes

## Type of change


## Content

Ensure keyboard opens for reply and text formatting modes.

Note that it doesn't seem possible to use `LocalSoftwareKeyboardController` to control the keyboard when the focused UI is an `AndroidView`.

## Motivation and context

- https://github.com/vector-im/element-x-android/issues/1315

## Screenshots / GIFs


[Screen_recording_20230914_185152.webm](https://github.com/vector-im/element-x-android/assets/4940864/daa6c597-312b-4a51-962d-9029b39c027c)

## Tests



- Close keyboard
- Reply to a message
- Check keyboard is open

- Close keyboard
- Open text formatting menu
- Check keyboard is open

## Tested devices

- OS version(s): 13

## Checklist
